app/stories/pix-collapsible.stories.js

```diff
@@ -57,19 +60,39 @@ export const collapsibleWithBlockTitle = (args) => {
   };
 };
 
+export const collapsible = Template.bind({});
+collapsible.args = {
+  title: 'Titre du contenu à dérouler en cliquant',
+};
```

Comment:
On perd l'exemple de l'icône dans la doc du collapsible

```suggestion
collapsible.args = {
  title: 'Titre du contenu à dérouler en cliquant',
  iconName: 'users',
};
```

<img width="470" alt="Capture d’écran 2024-10-10 à 18 22 47" src="https://github.com/user-attachments/assets/302fa8de-7771-4029-89cc-87febe2d0898">
<img width="470" alt="Capture d’écran 2024-10-10 à 18 23 06" src="https://github.com/user-attachments/assets/6d30a051-c5a5-4090-b663-f3959f1a6da6">